src/controller.ts

```diff
@@ -287,11 +287,12 @@ export class PhpStanController {
 
   protected makeCommandPath(cwd: string) {
     let binary = "";
-    if (process.platform === "win32") {
-      binary = path.resolve(cwd, "vendor/bin/phpstan.bat");
-    } else {
-      binary = path.resolve(cwd, "vendor/bin/phpstan");
-    }
+    let bindir = "vendor/bin";
+    let basename = process.platform === "win32" ? "phpstan.bat" : "phpstan";
+    try {
+        bindir = child_process.execSync("composer config bin-dir", {cwd, encoding: "utf-8"}).trim();
```

Comment:
The child_process.execSync return type 'Buffer', which property 'trim' does not exist on. You should convert 'Buffer' to 'String'.

Reply:
If you specify an encoding other than `buffer` (the default), the returned value is a String, not a Buffer.

Reply:
However, `tsc` can not resolve it 😂.
In your local try `npm run vscode:prepublish`.
See more details from [Pull Request #8 ](https://travis-ci.com/breeze2/vscode-phpstan/builds/135810752).

Reply:
Right... I had to fix it quickly on my environment and I just tested it by editing the compiled JS.
Should be fixed now!